QuickDeployment/V2/QuickDeploymentDesktopReadme.md

```diff
@@ -90,6 +90,10 @@ If you would like to change the hostname of this VM, please do so **before** goi
 
 ### 3. Run the Set-QDEnvironment.ps1 Script
 
+:memo: **Note**
```

Comment:
```suggestion
> :memo: **Note**
>
```

Reply:
Thank you. Messing with TOC, Ctrl+z'd too many times :joy: